frontends/precinct-scanner/src/screens/scan_warning_screen.tsx

```diff
@@ -154,7 +162,7 @@ function UndervoteWarningScreen({
                 )}
                 {partiallyVotedContestNames.length > 0 && (
                   <span>
-                    You can vote for more people in{' '}
+                    You may still vote in{' '}
```

Comment:
I think "can vote for more people" is more specific about what they can actually still do than "may still vote"

Reply:
Agreed. Implementing with "candidates" instead of "people".

> You can vote for more candidates in 2 contests.